xml/System.IO/Path.xml

```diff
@@ -2023,6 +2023,22 @@ Unlike <xref:System.IO.Path.GetTempFileName%2A>, <xref:System.IO.Path.GetRandomF
 
 Paths are resolved by calling the <xref:System.IO.Path.GetFullPath%2A> method before calculating the difference. The method uses the default file path comparison for the current platform (<xref:System.StringComparison.OrdinalIgnoreCase?displayProperty=nameWithType> for Windows and MacOs, <xref:System.StringComparison.Ordinal?displayProperty=nameWithType> for Linux.
 
+## Examples
+ The following code shows how to call the <xref:System.IO.Path.GetRelativePath%2A> method.
+
+```vb
+		' "C:/Program Files/Microsoft" relative to "C:/Program Files/Common Files" is "../Microsoft"
+		Debug.Assert(("../Microsoft" = Path.GetRelativePath("C:/Program Files/Common Files", "C:/Program Files/Microsoft")))
```

Comment:
I usually don't see examples displaying behavior with assertions; write to console instead.
```suggestion
		Console.WriteLine(Path.GetRelativePath("C:/Program Files/Common Files", "C:/Program Files/Microsoft"))
```